proxyd/proxyd.go

```diff
@@ -416,6 +417,10 @@ func Start(config *Config) (*Server, func(), error) {
 			if bgcfg.ConsensusHA {
 				tracker.(*RedisConsensusTracker).Init()
 			}
+		} else if bgcfg.RoutingStrategy == MulticallRoutingStrategy {
+			log.Info("configuring routing multicall strategy for backend_group", "name", bgName, "routing_strategy", bgcfg.RoutingStrategy)
+		} else {
+			log.Info("no routing strategy defined for backend_group using default round robin", "name", bgName, "routing_strategy", bgcfg.RoutingStrategy)
```

Comment:
the default should be fallback, not round robin

Reply:
Fallback is where we sequentially request he backends one at a time for a response with no consensus. This is what the default is routing strategy is.  I called this round robin in the log, I can update it to fallback.

Reply:
Can we have a explicit "fallback" option with its const as well?
Config examples should be updated stating this new routing_strategy config, with default to fallback.

Reply:
Sure, we can have a config option that is explicitly is routing_strategy == fallback, and lack of the the routing_strategy will default to the fallback strategy. 

Happy to add that. 


I'd also propose that we rename the config "fallbacks" which is used for consensus routing strategy to something like consensus_fallbacks to reduce confusion between these configs. Happy to add that in a follow up ticket

Reply:
One `fallback` is a value for "routing_strategy" and the other is the key that holds the list of fallback backends that is meaningful only for `consensus`, right?

If so, its ok to keep the same name.

Also, let's make this config backwards compatible:
* Mark the legacy `consensus_aware` as deprecated with some comments indicating how it should be specified now
* If `consensus_aware` is set to true, it should implicit set "routing_strategy" to `consensus` and print a INFO log at initialization
* If both `consensus_aware` and `routing_strategy` are populated at same time, it should fail at initialization with a message indicating they are mutual exclusive and "routing_strategy" is the preferred way moving forward

Reply:
Do you want an explicit routing strategy value for fallback, in addition to fallback being the default behavior if none is provided?

Reply:
yes

the config "language" definition should be
routing_strategy = fallback | consensus_aware | multicall

with fallback being the default in cause its omitted

the [if/else](https://github.com/ethereum-optimism/infra/pull/26/files#diff-01f9fee1156d98d875869e58c791a9602554a9e06932e224dcd0567e4cdad7b3R430) used to pick the right strategy *should not* handle the fallback case as a catch-all else

Reply:
Provided a function to handle this logic, the catch-all will fail